config/default.yml

```diff
@@ -523,6 +523,16 @@ Style/MultilineOperationIndentation:
 Style/NumericLiterals:
   MinDigits: 5
 
+Style/OptionHash:
+  # Blacklist contains the default set of warnings to raise. If you do not
```

Comment:
The description is quite unclear. It should say something like "A list of parameter names that will be flagged by this cop.".